packages/flutter/lib/src/gestures/events.dart

```diff
@@ -419,7 +420,6 @@ class PointerHoverEvent extends PointerEvent {
     Offset delta = Offset.zero,
     int buttons = 0,
     bool obscured = false,
-    double pressure = 0.0,
     double pressureMin = 1.0,
```

Comment:
What about pressureMin/Max? Do these make any sense without pressure?

Reply:
In my opinion, it’s not “no pressure”, but pressure is 0. Pressure min and max are properties of the device, and make sense to be passed as-is.